test/powershell/Modules/Microsoft.PowerShell.Security/CertificateProvider.Tests.ps1

```diff
@@ -0,0 +1,138 @@
+Import-Module (Join-Path -Path $PSScriptRoot 'certificateCommon.psm1') -Force
```

Comment:
The only imports functions, the actual prep work is done in the `beforeall` of the feature tests and is skipped on non-windows platforms.